src/DatePicker/Calendar.js

```diff
@@ -297,10 +289,12 @@ class Calendar extends Component {
       },
     };
 
-    const weekTitleDayStyle = prepareStyles(styles.weekTitleDay);
-    const weekTitleStyle = prepareStyles(styles.weekTitle);
-    const calendarContainerStyle = prepareStyles(styles.calendarContainer);
-    const yearContainerStyle = prepareStyles(styles.yearContainer);
+    prepareStyles(styles.root);
```

Comment:
I don't follow the logic behind this change. As far as I know, `prepareStyles` isn't supposed to have any side-effect in production. The `callOnce` transformer do, but we only use it in development.